### ISSUE TITLE
Handle dynamic .graphql imports

### ISSUE THREAD

**Issue by vikiival** [kodadot/nft-gallery#1521]
### What happened?

Currently, for the dynamic query import, I use magick like 

```ts
const query = this.urlPrefix === 'rmrk' 
  ? await import('@/queries/nftStatsByIssuer.graphql')
  : await import('@/queries/unique/nftStatsByIssuer.graphql')
```

In future, we will have more chains so having something like
```ts
const query = await import(`@/queries/${this.urlPrefix}/nftStatsByIssuer.graphql`)
```

would be perfect

### Please reproduce in steps

the current problem is that I am unable to get it up and running

### Expected Behavior

I've expected ...

### Screenshots

Paste from your clipboard or upload screenshots

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jbienesdev** [comment]
Hi @vikiival , for this issue since the example uses dynamic import, the gql file needs to be unwrapped by the `default` key. Without unwrapping `default`, the linter was complaining. So it goes like this
![image](https://user-images.githubusercontent.com/17470909/146002693-97f7109b-1504-4010-b35f-ab73f82a48e6.png)

We can also try to do the ES5 way of requiring
![image](https://user-images.githubusercontent.com/17470909/146003029-2478ae5d-1c19-425b-8b64-e128beacbeab.png)

Tested this by changing `this.path` , both gave the expected results from the query. 😄

**Comment by vikiival** [comment]
> the gql file needs to be unwrapped by the default key. 

I am aware of that

> EDIT: @ module resolver works

I was unable to run this

**Comment by jbienesdev** [comment]
Can confirm that it's working on my local and is able to run the appropriate query correctly.

```javascript
(await import(`@/queries/${this.path}/nftStatsByIssuer.graphql`)).default
```

Maybe you're in a file that doesn't support dynamic imports?

**Comment by yangwao** [comment]
@vikiival still thing or closing?

**Comment by vikiival** [comment]
Implemented in b9b8178b98cff4c47994268f2597b3ed8a27319d
